test/jtreg-ext/requires/VMProps.java

```diff
@@ -499,15 +508,67 @@ protected String dockerSupport() {
            }
          }
 
+        log("dockerSupport(): returning isSupported = " + isSupported);
         return "" + isSupported;
     }
 
+    // Returns comma-separated file names for stdout and stderr.
+    private String redirectOutputToLogFile(String msg, ProcessBuilder pb, String fileNameBase) {
```

Comment:
It would be better to return List<String> fileNames rather to combine/parse the String.